support-frontend/assets/components/tabs/tabs.jsx

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import React, { type Node } from 'react';
+import { tabList, tabButton, tabPanel } from './tabsStyles';
+
+type TabElement = 'a' | 'button';
+
+type TabProps = {|
+  id: string,
```

Comment:
I'd rather keep the `id` prop generic- we only use the tabs component on the Paper page at the moment, but it would be nice to be able to just pick it up and use it elsewhere if necessary.